repository.owl

```diff
@@ -0,0 +1,29 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
```

Comment:
I have no interest in writing XML (I find it difficult to read and difficult to maintain), but if you'd like to convert this document, be my guest.